docker-compose.yml

```diff
@@ -1,7 +1,7 @@
 version: '2'
 services:
   unifi:
-    image: ryansch/unifi-rpi:v5
+    image: ryansch/unifi-rpi:latest
```

Comment:
I started this repo right around the v4/v5 change.  `latest` is what we actually want. 🎉